deployment/arm/deploy-region.sh

```diff
@@ -37,3 +39,12 @@ az group deployment create \
   --resource-group $resourceGroupName \
   --template-file region.json \
   --parameters location=$location apimName=$apimName appServicePlanName=$appServicePlanName frontendAppName=$frontendAppName backendAppName=$backendAppName appInsightsName=$appInsightsName
+
+cosmosEndPoint=`az group deployment show -g  rg-$globalScope -n Urlist-global-cosmosdb --query properties.outputs.documentEndpoint.value -o tsv`
```

Comment:
should we make cosmodb name configurable? we might end up deploying a new cosmosdb per hack

Reply:
Valid point, should we add the cosmosdb name as part of the variable group?

Reply:
Based on the discussion in planning today, we will need to configure names not only for cosmosdb but for all global resources, like frontdoor as well. I will create a task to configure all global resources names and will handle that in a separate PR.